apps/src/javalab/Javalab.js

```diff
@@ -76,6 +79,14 @@ Javalab.prototype.init = function(config) {
   const onContinue = this.onContinue.bind(this);
   const onCommitCode = this.onCommitCode.bind(this);
   const onInputMessage = this.onInputMessage.bind(this);
+  let visualization;
+  if (this.level.csaViewMode === 'neighborhood') {
```

Comment:
will there be multiple modes? we should capture these in a constant or enum somewhere (maybe a javalab/constants.js file like we have for other labs -- [applab/constants.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/constants.js))

Reply:
✔️